server/item/recipe/item.go

```diff
@@ -16,6 +16,51 @@ type Item interface {
 	Empty() bool
 }
 
+// inputItems ...
```

Comment:
```suggestion
// inputItem ...
```

Reuse this type for `inputItems`